frontend/src/App.js

```diff
@@ -16,8 +17,9 @@ class App extends Component {
           <Route exact path="/" component={Home} />
           <Route exact path="/register" component={Register} />
           <Route exact path="/login" component={Login} />
-          <Route path="/kids" component={KidsPage} />
-          <Route path="/parents" component={ParentsPage} />
+          <Route exact path="/kids" component={KidsPage} />
+          <Route exact path="/parents" component={ParentsPage} />
+          <Route exact path="/parents/advice" component={ParentAdvice} />
```

Comment:
I added a comment on the `ParentsPage` file -- basically, we can remove the `ParentAdvice` route